packages/gboost-ui/src/components/Fields/FileUploadField.tsx

```diff
@@ -78,6 +78,7 @@ export function _FileUploadField(
         style={{ display: "none" }}
         type="file"
         onChange={onChange || uncontrolledOnChange}
+        onClick={(e) => (e.currentTarget.value = "")}
```

Comment:
could you add a comment above this line with why we're doing this and maybe the SO post?